src/Options/Factory.hpp

```diff
@@ -26,24 +27,17 @@
 /// input file.
 ///
 /// To use a factory, the base class (here `BaseClass`) should inherit
-/// from Factory<BaseClass> and define a static method
-/// \code
-///  static std::string class_id() { return "BaseClass"; }
-/// \endcode
-/// and a type
+/// from Factory<BaseClass> and define a type
 /// \code
 ///  using creatable_classes = tmpl::list<Derived1, ...>;
 /// \endcode
 ///
 /// Derived classes should:
-/// 1) define a static member function std::string class_id() which
-///    returns a unique id for the derived class, similar to the base
-///    class method shown above.
-/// 2) define static OptionString_t help containing class-specfic help
+/// 1) define static OptionString_t help containing class-specfic help
```

Comment:
doxygen does have a syntax for [enumerated list](https://www.stack.nl/~dimitri/doxygen/manual/lists.html) (having to relabel the comments made me think of this...)